File: ProyectoClipMoney2020/Models/Clases/Cliente.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProyectoClipMoney2020.Models
{
    public class Cliente
    {
        public long idCliente { get; set; }
        public string nombre { get; set; }
        public string apellido { get; set; }
        public DateTime fechaNacimiento { get; set; }
        public TipoDocumento tipoDocumento { get; set; }
        public int nroDocumento { get; set; }
        public string fotoFrenteDocumento { get; set; }
        public string fotoDorsoDocumento { get; set; }
        public string email { get; set; }
        public long telefono { get; set; }
        public Domicilio domicilio { get; set; }
        public Nacionalidad nacionalidad { get; set; }
        public string passEncriptada { get; set; }
        public string usuario { get; set; }
        public SituacionCrediticia situacionCrediticia { get; set; }
        public IList<Cuenta> cuentas { get; set; }

        public string ObtenerFecha()
        {
            return fechaNacimiento.ToString("yyyy-MM-ddTHH:mm:ss.fff");
        }
    }
}

File: app/src/app/Modelos/Cliente.ts
import {TipoDocumento} from './TipoDocumento';
import {Domicilio} from './Domicilio';
import {Nacionalidad} from './Nacionalidad';
import {SituacionCrediticia} from './SituacionCrediticia';
import {Cuenta} from './Cuenta';


export class Cliente {
    IdCliente: number;
    Nombre: string;
    Apellido: string;
    FechaNacimiento: string;
    TipoDocumento: TipoDocumento;
    NroDocumento: number;
    FotoFrenteDocumento: string;
    FotoDorsoDocumento: string;
    Email: string;
    Telefono: number;
    Domicilio: Domicilio;
    Nacionalidad: Nacionalidad;
    PassEncriptada: string;
    Usuario: string;
    SituacionCrediticia: SituacionCrediticia;
    Cuentas: Cuenta[];
  }

